docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/deployment.md

```diff
@@ -241,8 +250,77 @@ Follow these instructions to update devices.
     [{"apiVersion":"v3","statusCode":201,"id":"fb5fb7f2-768b-4298-a916-d4779523c6b5"}]
     ```
     
-2. Map credentials using the `map-credentials.sh` script.  
+2. Map the credentials for the camera.
+
+!!! note
+    The preferred method for setting the credentials is through the use of the curl commands.
+
+=== "Curl Command"
+    !!! note
+        If running in secure mode all the api executions need the [JWT token](#token-generation) generated previously.
+
+    a. Check the status of the credentials map.
+    ```bash
+    curl -H "X-Consul-Token:<consul-token>" -X GET "http://localhost:8500/v1/kv/edgex/v3/device-onvif-camera/AppCustom/CredentialsMap?keys=true" | jq .
+    ```
+    Example response:
+    ```bash
+    [
+    "edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/NoAuth",
+    "edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/credentials001",
+    "edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/credentials002"
+    ]
+    ```
```

Comment:
i would move this section to right before `h.`, since technically these are just the mappings and not all the available secret names.